react-example/src/style.css

```diff
@@ -0,0 +1,5 @@
+h1,
```

Comment:
do we need to add these styles? I noticed we don't have any styles in the vue example and then we have different styles for angular, we should probably make the examples all consistent and looking the same, but that can be an improvement task for later

Reply:
yeah styles aren't needed. We can update the Angular one in a different PR